spark-native/scripts/spark/added/launch.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# spark likes to be able to lookup a username for the running UID, if
+# no name is present fake it.
+cat /etc/passwd > /tmp/passwd
+echo "$(id -u):x:$(id -u):$(id -g):dynamic uid:$SPARK_HOME:/bin/false" >> /tmp/passwd
+
+export NSS_WRAPPER_PASSWD=/tmp/passwd
+# NSS_WRAPPER_GROUP must be set for NSS_WRAPPER_PASSWD to be used
+export NSS_WRAPPER_GROUP=/etc/group
+
+export LD_PRELOAD=libnss_wrapper.so
+
+# If SPARK_MASTER_ADDRESS env varaible is not provided, start master,
+# otherwise start worker and connect to SPARK_MASTER_ADDRESS
+if [ -z ${SPARK_MASTER_ADDRESS+_} ]; then
```

Comment:
kube native is not actually using `launch.sh`, so this file could be removed.  the current "P1" branch (i.e. the original "foxish" based one) uses `driver.sh` and `executor.sh`

That said, I'm gravitating toward the proposed scheme of the kube-native code just invoking a driver image and executor images, which are assumed to have their own CMD embedded.  It will be easier to reason about for people spinning up custom images, and will allow them to specify their own startup logic, without being concerned with some particular command structure coming from the pod building API.